hazelcast/src/test/resources/hazelcast-fullconfig-without-network.yaml

```diff
@@ -56,7 +56,7 @@ hazelcast:
       batch-publisher:
         tokyoPublisherId:
           cluster-name: tokyo
-          class-name: com.hazelcast.enterprise.wan.impl.replication.WanBatchReplication
+          class-name: com.hazelcast.enterprise.wan.impl.replication.WanBatchPublisher
```

Comment:
I think this line can be removed. There is no code handling `class-name` in the config processor and there is no `class-name` in the xsd either. The full-example yaml is fine.

Reply:
Done